client/src/components/Questions.jsx

```diff
@@ -12,19 +12,54 @@ const Body = styled.div `
 `
 const QuestionModule = styled.div `
   margin-bottom: 12px;
+  background-color: rgb(255, 255, 255);
 `
 const Question = styled.div `
-  padding: 16px 24px 12px;
+  padding: 16px 24px 24px 12px;
   border: solid #e0e0e0;
   border-width: 0 0 1px;
 `
-// const Answerss = styled.div `
-//   padding: 16px 24px 12px;
-//   border: solid #e0e0e0;
-//   border-width: 0 0 1px;
-// `
+
+const QuestionHeader = styled.div `
```

Comment:
Are you planning on adding any styling to this one or any of the other empty styled.divs? If not I think you can use a regular div!

Reply:
Yeah I wasn't sure. If I don't I'll make sure to switch it back to a regular div. Thanks for the catch!